utils/utils/src/main/scala/pl/touk/nussknacker/engine/util/json/ToJsonEncoder.scala

```diff
@@ -66,7 +67,7 @@ case class ToJsonEncoder(
           case a: UUID           => safeString(a.toString)
           case a: DisplayJson    => a.asJson
           case a: scala.collection.Map[_, _] => encodeMap(a.toMap)
-          case a: java.util.Map[_, _]        => encodeMap(a.asScala.toMap)
+          case a: java.util.Map[_, _]        => encode(ListMap(a.asScala.toList: _*))
```

Comment:
It might by not optimal. You should check what is done in implementation. Maybe there is some better approach when we assume that given map is a `LinkedHashMap`? Can you write a micro benchmark for that and verify the difference in times between old serialization method vs the new one?